docs/user/index.rst

```diff
@@ -0,0 +1,120 @@
+######################
+Using Aldryn Reversion
+######################
+
+After you set up and configured your models (they are registered for revisions
+support) you can start using ``aldryn-reversion`` end-user features.
+
+************************
+Restoring object version
+************************
+
+Most likely you already know that django keeps track of object changes also
+known as history, it is available on object admin edit form on the top of the
+page. With ``aldryn-reversion`` that history has more features.
+As soon as object is created - 'Initial version' is created and treated as
+restore point.
+
+Each record in history has a date which is now a link to restore revision
+form.
+When you need to restore any object version you just need to click on desired
+date, check what will be changed  and either restore that revision or go back
+to object history.
+
+.. note::
+    Be aware that in current implementation revision is restored as is.
```

Comment:
How about:
"Note that the revision will be restored 'as is' - all objects related to the object you are restoring will also be restored to the same revision."
?